train.py

```diff
@@ -1,12 +1,17 @@
+import csv
 import random
 
 from autowebcompat import network, utils
 
 labels = utils.read_labels()
 
+def update_all_image_names(images):
+    return [i for i in images if i in labels]
+
 utils.prepare_images()
 all_image_names = utils.get_images()
```

Comment:
I think the filtering should be done here, otherwise `all_images` will contain images that are not actually used.